lib/Extractor/Solver.cpp

```diff
@@ -214,18 +215,21 @@ class BaseSolver : public Solver {
     if(SMTSolver->supportsModels()) {
       if (EnableExhaustiveSynthesis) {
         ExhaustiveSynthesis ES;
-        EC = ES.synthesize(SMTSolver.get(), BPCs, PCs, LHS, RHS, IC, Timeout);
-        if (EC || RHS)
+        EC = ES.synthesize(SMTSolver.get(), BPCs, PCs, LHS, RHSs,
```

Comment:
did you intend to change this line? was it too long?

Reply:
RHS is changed to RHSs, and it is too long so I wrapped.